Sustainability/SustainabilityProfileIntro.md

```diff
@@ -0,0 +1,121 @@
+
+# Contents
+[add_toc]
+
+# Overview
+
+This document contains the Redfish interface requirements for reporting Sustainability data
+
+Profile source: OCP-PDU-Profile.json
+
+Direct feedback to: jeff.autor@hpe.com
+
+
+# Sustainability Use Cases
+
+The purpose of this profile is to ensure that common desired software use cases can be achieved using the standard API and data model contents provided by the device.  These common use cases are described below, and utilize the Redfish resources shown in the Profile Reference Guide section of this document.  Portions of the JSON payload responses are shown as examples, and are the result of HTTP GET operations performed on the supported URIs shown in the reference sections. 
+
```

Comment:
Add statement about "scope" of the profile - this covers any datacenter gear except for the power distribution (PDU and above) equipment.  That equipment will have the same data, in the same formats, but in different locations (because it's the primary function of the equipment).  And that gear will cover the same topics from their own profile requirements (regardless of sustainability needs).